src/XenobladeChroniclesX/Mods/BattleEscapeDistance/rules.txt

```diff
@@ -1,16 +1,33 @@
 [Definition]
 titleIds = 0005000010116100,00050000101C4C00,00050000101C4D00
-name = "Escape distance reducer"
-path = "Xenoblade Chronicles X/Mods/Battle/Escape distance reducer"
-description = Reduce escape distance needed to lose aggro from enemies.
-version = 5
+name = "Enemy aggro reducer"
+path = "Xenoblade Chronicles X/Mods/Battle/Enemy aggro reducer"
```

Comment:
Does this mod change how far the enemies will start being aggro at, or does it change the escape distance? Isn't the previous name more descriptive? Maybe change it to Enemy escape distance reducer?

Reply:
think I am going to change it to "enemy de-aggro range reducer" as that seems to be the best description I can think of